### DISCUSSION TITLE
Can SK6812 WWA be supported?

### DISCUSSION THREAD

**Discussion by ashp8i** [Makuna/NeoPixelBus#552] [Q&A] [answered]
Not sure how straight forward this might be but I can use these strips pretending they are RGB

**Comment by Makuna** [comment, marked as answer]
What do you mean WWA?  I do find this in reference to White, Warm White, Amber as three elements per pixel but is this what you mean?

NeoSk6812Method is the method.
Using the NeoRgbFeature would map a RgbColor with R to White, G to Warm White, and B to Amber; that is if that is the true color order of WWA.  A little experimentation should tell you the actual order.

**Reply by ashp8i** [reply]
Thats correct I would like to find out if this is sufficient for projects like esphome or WLED can control Warm White, Cool White and Amber SK6812 strips are there any upstream changes required?

**Reply by Makuna** [reply]
Did you try it?  

There shouldn't be but note that some of these Solutions directly write into the buffer which ignores the features job of translation, but this would only affect the color order.

Now how they handle everything above that can correctly handle whites instead of colors is the question.  You would have to ask them; but I suspect it should just work.

**Comment by ashp8i** [comment]
Many thanks for confirming
________________________________
